schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/mail/DefaultMailService.java

```diff
@@ -115,8 +115,9 @@ private boolean send(InternetAddress fromAddress, InternetAddress[] toAddress, I
         Preconditions.checkNotNull(toAddress);
         Preconditions.checkNotNull(bccAddress);
         Preconditions.checkNotNull(subject);
-        Preconditions.checkNotNull(htmlBody);
-        Preconditions.checkNotNull(textBody);
+        if (htmlBody == null && textBody == null) {
```

Comment:
Previously we had emails that always had the html and text set. I introduced some emails that have only html or text set.